samples/librats-server/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.5.1)
+project(librats-server)
+
+# /usr/local
+set(RATS_INSTALL_PATH "/usr/local")
+# include/rats
+set(RATS_INSTALL_INCLUDE_PATH "${RATS_INSTALL_PATH}/include/")
+
+# Set include directory
+set(INCLUDE_DIRS ${RATS_INSTALL_INCLUDE_PATH})
+include_directories(${INCLUDE_DIRS})
+
+# lib/rats
+set(RATS_INSTALL_LIB_PATH "${RATS_INSTALL_PATH}/lib/librats")
+set(LIBRARY_DIRS ${RATS_INSTALL_LIB_PATH}
+                 ${RATS_INSTALL_LIB_PATH}/verifiers
+                 ${RATS_INSTALL_LIB_PATH}/attesters
+                 )
```

Comment:
Maybe there is no need to set ${RATS_INSTALL_LIB_PATH}/verifiers and  ${RATS_INSTALL_LIB_PATH}/attesters to LIBRARY_DIRS